flask_common/mongo/fields/crypto.py

```diff
@@ -34,6 +35,12 @@ def __init__(self, key_or_list, *args, **kwargs):
             assert len(key) == KEY_LENGTH, 'invalid key size'
         return super(EncryptedStringField, self).__init__(*args, **kwargs)
 
+    def __set__(self, instance, value):
+        # Handle unicode strings by encoding them
+        if PY3 and isinstance(value, txt_type):
```

Comment:
Why do we only do this for Py3? Shouldn't py2's unicode strings be encoded into utf-8 bytes as well?